src/rpc/mempool.cpp

```diff
@@ -812,6 +815,104 @@ static RPCHelpMan savemempool()
     };
 }
 
+static std::vector<RPCResult> OrphanDescription()
+{
+    return {
+        RPCResult{RPCResult::Type::STR_HEX, "txid", "The transaction hash in hex"},
+        RPCResult{RPCResult::Type::STR_HEX, "wtxid", "The transaction witness hash in hex"},
+        RPCResult{RPCResult::Type::NUM, "size", "The serialized transaction size"},
```

Comment:
maybe "bytes" instead? to make it hard to confuse with vsize

Reply:
"bytes" seems clearer to me as well. Updated.

Originally "size" was chosen because it aligns with `getrawtransaction`.
If there's friction with using "bytes", I can change it back to "size".

> Argument and field naming: please consider whether there is already a naming style or spelling convention in the API for the type of object in question (blockhash, for example), and if so, try to use that

https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#rpc-interface-guidelines